### ISSUE TITLE
Charts library does not work in internet explorer

### ISSUE THREAD

**Issue by brfreek** [mendixlabs/charts#20]
Charts stay at loading screen and show the following error in the console log:
SCRIPT1003: Expected ':'
chunk0.js (1,234968)
```

Unhandled promise rejection Error: Loading chunk 0 failed.
   "Unhandled promise rejection"
   {
      [functions]: ,
      __proto__: { },
      description: "Loading chunk 0 failed.",
      message: "Loading chunk 0 failed.",
      name: "Error",
      stack: "Error: Loading chunk 0 failed.
   at a.sent (https://charts102.mxapps.io/widgets/com/mendix/widget/custom/AreaChart/AreaChart.js?636549163634606700:1:6445)
   at Anonymous function (https://charts102.mxapps.io/widgets/com/mendix/widget/custom/AreaChart/AreaChart.js?636549163634606700:1:9853)
   at Anonymous function (https://charts102.mxapps.io/widgets/com/mendix/widget/custom/AreaChart/AreaChart.js?636549163634606700:1:7292)
   at Anonymous function (https://charts102.mxapps.io/widgets/com/mendix/widget/custom/AreaChart/AreaChart.js?636549163634606700:1:6636)
   at s (https://charts102.mxapps.io/widgets/com/mendix/widget/custom/AreaChart/AreaChart.js?636549163634606700:1:6220)
   at a (https://charts102.mxapps.io/mxclientsystem/mxui/mxui.js?636549163634606700:7:596581)
   at Anonymous function (https://charts102.mxapps.io/mxclientsystem/mxui/mxui.js?636549163634606700:7:596705)
   at c (https://charts102.mxapps.io/mxclientsystem/mxui/mxui.js?636549163634606700:7:600201)",
      Symbol(observable)_l.6nmn6z0chh3: undefined,
      Symbol(react.coroutine)_h.6nmn6z0chh3: undefined,
      Symbol(react.element)_g.6nmn6z0chh3: undefined,
      Symbol(react.portal)_j.6nmn6z0chh3: undefined,
      Symbol(react.yield)_i.6nmn6z0chh3: undefined,
      Symbol(rxSubscriber)_k.6nmn6z0chh3: undefined
   }

```

**Comment by brfreek** [comment]
This is for all charts in the library, not just the AreaChart

**Comment by Andries-Smit** [comment]
This issue is resolved in version 1.0.0 beta 5
https://appstore.home.mendix.com/link/app/105695/
